src/compiler/sys.ts

```diff
@@ -1559,10 +1559,13 @@ namespace ts {
             }
 
             function readFileWorker(fileName: string, _encoding?: string): string | undefined {
-                if (!fileExists(fileName)) {
+                let buffer: Buffer;
+                try {
+                    buffer = _fs.readFileSync(fileName);
+                }
+                catch (e) {
                     return undefined;

```

Comment:
Should errors other than `EEXIST` be rethrown?  (Seems worse, but that's what it seems to have done before.)

Reply:
Oh, I see from the bug that swallowing the exception is a feature of this change.